R/weather-norms.R

```diff
@@ -423,6 +422,7 @@ weather_norms_area <- function(polygon
     })
   }
 
+  cat(paste0('Creating aWhere Raster Grid within Polygon\n'))
```

Comment:
This should be made a helper fxn because it will be the exact same for the other endpoints